$OpenBSD: patch-wafadmin_Tools_ccroot_py,v 1.3 2010/06/01 12:47:10 dcoppa Exp $

Don't create symlink for shared libraries versions.
Use good version numbers for libs.

--- wafadmin/Tools/ccroot.py.orig	Sun Mar 14 17:57:48 2010
+++ wafadmin/Tools/ccroot.py	Sun Mar 14 17:59:04 2010
@@ -580,7 +580,7 @@ def apply_vnum(self):
 		name2 = libname + '.' + nums[0]
 
 	if self.env.SONAME_ST:
-		v = self.env.SONAME_ST % name2
+		v = self.env.SONAME_ST % name3
 		self.env.append_value('LINKFLAGS', v.split())
 
 	bld = self.bld
@@ -590,13 +590,13 @@ def apply_vnum(self):
 	if not path: return
 
 	bld.install_as(path + os.sep + name3, node, env=self.env)
-	bld.symlink_as(path + os.sep + name2, name3)
-	bld.symlink_as(path + os.sep + libname, name3)
+	#bld.symlink_as(path + os.sep + name2, name3)
+	#bld.symlink_as(path + os.sep + libname, name3)
 
 	# the following task is just to enable execution from the build dir :-/
 	tsk = self.create_task('vnum')
 	tsk.set_inputs([node])
-	tsk.set_outputs(node.parent.find_or_declare(name2))
+	tsk.set_outputs(node.parent.find_or_declare(name3))
 
 def exec_vnum_link(self):
 	path = self.outputs[0].abspath(self.env)
